shared/getting-started/getDeviceOnDash/orangepi-plus2.md

```diff
@@ -1,6 +1,6 @@
 Insert the SD card into your **{{ $device.name }}** and, if necessary, connect the ethernet cable or the USB WiFi adapter. Now power up your **{{ $device.name }}** by inserting the micro USB cable.
 
-It will take a minute or two for the **{{ $device.name }}** to appear on your {{ $names.company.lower }} [dashboard][resinDash]. While you wait, the resinOS is expanding the partitions on your SD card to use all available space, installing a custom Linux environment, and establishing a secure connection with the resin.io servers.
+It will take a minute or two for the **{{ $device.name }}** to appear on your {{ $names.company.lower }} [dashboard][resinDash]. While you wait, the balenaOS is expanding the partitions on your SD card to use all available space, installing a custom Linux environment, and establishing a secure connection with the balena servers.
```

Comment:
```suggestion
It will take a minute or two for the **{{ $device.name }}** to appear on your {{ $names.company.lower }} [dashboard][resinDash]. While you wait, the {{ $names.os.lower }} is expanding the partitions on your SD card to use all available space, installing a custom Linux environment, and establishing a secure connection with the {{ $names.company.lower }} servers.
```